tests/test_directory_sync.py

```diff
@@ -43,18 +46,144 @@ def mock_users(self):
                         ],
                     },
                     "id": "directory_usr_id",
-                }
+                    "object": "directory_user",
```

Comment:
Thought on the tests - is there a way in Python to generate a large amount of objects so we can fully test the auto pagination?

Reply:
I'll look in to this 👀 I think you're right that we can improve on how these tests are structured with a larger mock data set.